Tasks/Common/codecoverage-tools/codecoverageconstants.ts

```diff
@@ -26,9 +26,9 @@ subprojects {
 
         reports {
             html.enabled = true
-            html.destination "\${buildDir}/jacocoHtml"
+            html.destination file("\${buildDir}/jacocoHtml")
```

Comment:
It looks like the `$` is being escaped here, so the end result would be `path/to/build.gradle/${buildDir}/jacocoHtml`, rather than replacing `${buildDir}` with it's value.  Or am I missing something?

Reply:
This is intended to be replace in the build.gradle file instead, as a project property.

Reply:
Oh I see now, phew that's confusing